plugins/comments.md

```diff
@@ -54,20 +66,13 @@ tinymce.init({
 })
 ```
 
-Optional values: addcomment, showcomments
+**Optional values:** `addcomment`, `showcomments`
```

Comment:
I think this section is confusing. Maybe just consider deleting it (Optional Values and Result for the codepen).

Reply:
Fixed